rllib/policy/eager_tf_policy.py

```diff
@@ -261,15 +261,16 @@ def __init__(self, observation_space, action_space, config):
 
         @override(Policy)
         def postprocess_trajectory(self,
-                                   samples,
+                                   sample_batch,
```

Comment:
Should call these `rollout_fragment` probably from now on, though this would be a large renaming.